powerfulseal/policy/ps-schema.yaml

```diff
@@ -153,6 +153,14 @@ definitions:
             items:
               oneOf:
               - "$ref": "#/definitions/filterPropertyNode"
+          retries:
+            type: object
+            description: >
+              An object of retry criteria to rerun set actions
+            items:
```

Comment:
Hey, as far as I know, there should no longer be `items` here, like [here](https://github.com/powerfulseal/powerfulseal/blob/570d8b0921c35ccf4bfe0ee4a465d93d414290c8/powerfulseal/policy/ps-schema.yaml#L305-L309).

Could you please add the extra things to the kitchen sink validation test in [here](https://github.com/powerfulseal/powerfulseal/blob/570d8b0921c35ccf4bfe0ee4a465d93d414290c8/tests/policy/example_config.yml). This way it will get validated in the CI, and we'll know for sure.

Reply:
I tried to add both ways you can have retries (timeout and count) into different parts of the example_config. Is this ok? It's a little different because I couldn't make a list of retries like all the other areas